mtgjson5/providers/tcgplayer.py

```diff
@@ -478,6 +479,13 @@ def generate_mtgjson_sealed_product_objects(
 
         sealed_data = get_tcgplayer_sealed_data(group_id)
 
+        if wc_set:
+            correct_data = []
```

Comment:
`correct_data = [product for product in sealed_data if set_code[:-2] in product["cleanName"]`